src/components/Thread.vue

```diff
@@ -423,4 +423,9 @@ export default {
 .user-bubble__title {
 	cursor: pointer;
 }
+.emptycontent {
+	display: flex;
+	position: relative;
+	margin-top: 50vh;
+}
```

Comment:
isn't there an off-the-shelf component for this?

Reply:
yes, the loading component, the styling is slightly off here. If i dont add any styling it will be aligned on the left.

Reply:
Can we use that component? I doubt that the margin-top:50vh is responsive across window sizes

Reply:
we are using it, see on the template